easybuild/easyconfigs/o/OpenCV/OpenCV-3.1.0-foss-2016a.eb

```diff
@@ -11,8 +11,16 @@ description = """OpenCV (Open Source Computer Vision Library) is an open source
 
 toolchain = {'name': 'foss', 'version': '2016a'}
 
-source_urls = ['https://github.com/Itseez/opencv/archive/']
-sources = ['%(version)s.zip']
+source_urls = [
+    'https://github.com/Itseez/opencv/archive/',
+    #The Hash is version dependent! see 3rdparty/ippicv/downloader.cmake
+    'https://raw.githubusercontent.com/Itseez/opencv_3rdparty/81a676001ca8075ada498583e4166079e5744668/ippicv',
+]
+
+sources = [
+    '%(version)s.zip',
+    ('ippicv_linux_20151201.tgz', 'cp %s %(builddir)s'),
```

Comment:
@boegel Actually it worked for me. I have checked in the builddir, and the file was there.